.github/workflows/main.yml

```diff
@@ -188,6 +188,7 @@ jobs:
         shell: bash -l {0}
         run: |
           pybind11-stubgen libmambapy.bindings
+          pre-commit run --files stubs/libmambapy/binginds-stubds/__init__.pyi
```

Comment:
Is this a typo? binginds-stubds -> bindings-stubs?

Reply:
Yes definitely! Tahnks for catching it